tests/tests/validator/PresentTest.php

```diff
@@ -28,6 +28,9 @@ public function data_validate()
 			array(new Jam_Range('|'), FALSE, TRUE),
 			array(new Jam_Range('1|'), FALSE, TRUE),
 			array(new Jam_Range('|1'), FALSE, TRUE),
+			array(new Jam_Price(1), FALSE, TRUE),
+			array(new Jam_Price(0), FALSE, FALSE),
+			array(new Jam_Price(0), TRUE, TRUE),
```

Comment:
These tests would require `openbuildings/jam-monetary` to be listed as a dev dependency.